bodo/tests/test_df_lib/test_end_to_end.py

```diff
@@ -49,9 +49,9 @@ def test_from_pandas(datapath, index_val, set_stream_batch_size_three):
     with temp_config_override("dataframe_library_run_parallel", False):
         bdf = bd.from_pandas(df)
         assert bdf.is_lazy_plan()
-        assert bdf.plan is not None
-        assert bdf.plan.plan_class == "LogicalGetPandasReadSeq"
-        duckdb_plan = bdf.plan.generate_duckdb()
+        assert bdf._mgr._plan is not None
```

Comment:
This check is effectively the same as the previous line.  Perhaps these should be switched to use the ExecState so it is clearer.

Reply:
Removed it. The ExecState check is also equivalent in `_exec_state` (`if self.is_lazy_plan(): return ExecState.PLAN`).